### PR TITLE
style: dot-notation import/first no-undef

### PR DESCRIPTION
**Pull request by mightyiam**

### CODE REVIEW

File: perf/benchmarks.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+// Had not been maintained for a while.
```
Comment (paldepind) [inline perf/benchmarks.js]: I think it should say "Has not ..."

File: src/test/dataset.ts [hunk 2]
```diff
@@ -2,10 +2,10 @@ import assert from 'assert'
 
 import datasetModule from '../modules/dataset'
 import { init } from '../snabbdom'
+import h from '../h'
```
Comment (paldepind) [inline src/test/dataset.ts]: Linebreak between imports and code?

File: src/test/core.ts [hunk 3]
```diff
@@ -5,16 +5,16 @@ import { init } from '../snabbdom'
...
-  classModule,
-  propsModule,
-  eventListenersModule
-]);
 import h from '../h'
 import toVNode from '../tovnode'
 import vnode, { VNode } from '../vnode'
 import htmlDomApi from '../htmldomapi'
 import { CreateHook, InsertHook, PrePatchHook, RemoveHook, InitHook, DestroyHook, UpdateHook } from '../hooks'
+var patch = init([
```
Comment (paldepind) [inline src/test/core.ts]: Linebreak between imports and code?

File: src/test/eventlisteners.ts [hunk 4]
```diff
@@ -3,10 +3,10 @@ import { VNode } from '../vnode';
 
 import { init } from '../snabbdom'
 import eventListenersModule from '../modules/eventlisteners'
+import h from '../h'
 var patch = init([
   eventListenersModule
 ]);
-import h from '../h'
```
Comment (paldepind) [inline src/test/eventlisteners.ts]: Linebreak between imports and code?

File: src/test/thunk.ts [hunk 5]
```diff
@@ -1,11 +1,11 @@
 import assert from 'assert'
 
 import { init } from '../snabbdom'
-var patch = init([
-]);
 import h from '../h'
 import thunk from '../thunk'
 import { VNode } from '../vnode';
+var patch = init([
```
Comment (paldepind) [inline src/test/thunk.ts]: Linebreak between imports and code?

File: .eslintrc.js [hunk 6]
```diff
@@ -6,6 +6,7 @@ module.exports = {
     '/dist/'
   ],
   rules: {
+    'import/newline-after-import': 'error',
```
Comment (mightyiam) [inline .eslintrc.js]: @paldepind :wink:

**Comment by mightyiam** [comment]
Build on master failed after I merged this in. Not sure how that could happen after the CI run passed here... Anyone got a clue?

**Comment by mightyiam** [comment]
So I force-pushed master to one commit earlier, which undid this merge and am doing it again in #548 .

**Pull request merged**